### PR TITLE
Update direct-routing-plan.md

### PR DESCRIPTION
**Pull request by NMuravlyannikov**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -112,6 +112,14 @@ For more information about Phone System licensing, see [Get the most from Office
 
 For more information about Phone System licensing, see [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md). 
 
+## Supported end points
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
## Supported endpoints 
```

File: Teams/direct-routing-plan.md [hunk 2]
```diff
@@ -112,6 +112,14 @@ For more information about Phone System licensing, see [Get the most from Office
 
 For more information about Phone System licensing, see [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md). 
 
+## Supported end points 
+
+You can use as end point:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
The following can be used as endpoints:
```

File: Teams/direct-routing-plan.md [hunk 3]
```diff
@@ -112,6 +112,14 @@ For more information about Phone System licensing, see [Get the most from Office
...
+- Any Teams cleintg.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Any Teams client. 
```

File: Teams/direct-routing-plan.md [hunk 4]
```diff
@@ -112,6 +112,14 @@ For more information about Phone System licensing, see [Get the most from Office
 
 For more information about Phone System licensing, see [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md). 
 
+## Supported end points 
+
+You can use as end point:
+- Any Teams cleintg. 
+- Common Area Phones, plese see [Set up the Common Area Phone license for Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/set-up-common-area-phones). Note you do not need Calling Plan license when setting the Common Area Phone with Direct Routing.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Common Area Phones. Please see [Set up the Common Area Phone license for Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/set-up-common-area-phones) for more information. Note that you do not need a Calling Plan license when setting up a Common Area Phone with Direct Routing.
```

File: Teams/direct-routing-plan.md [hunk 5]
```diff
@@ -112,6 +112,14 @@ For more information about Phone System licensing, see [Get the most from Office
...
+- Skype for Business 3PIP phones. Please read [Skype for Business phones (3PIP) support with Microsoft Teams](https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Skype-for-Business-phones-3PIP-support-with-Microsoft-Teams/ba-p/789351)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Skype for Business 3PIP phones. Please read [Skype for Business phones (3PIP) support with Microsoft Teams](https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Skype-for-Business-phones-3PIP-support-with-Microsoft-Teams/ba-p/789351).
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**